<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>sensor1</class>
 <widget class="QWidget" name="sensor1">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1200</width>
    <height>800</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
    <horstretch>255</horstretch>
    <verstretch>255</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>1200</width>
    <height>800</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1200</width>
    <height>800</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>温度传感器数据</string>
  </property>
  <widget class="QLabel" name="bak">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1200</width>
     <height>800</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>1200</width>
     <height>775</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>1600</width>
     <height>800</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">#bak{
	
	background-image: url(:/bak/bk4.png);
	background-repeat : no-repeat;
	background-position:  center;
	background-size: cover;
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="time">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>100</y>
     <width>1181</width>
     <height>61</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>1181</width>
     <height>61</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>1181</width>
     <height>61</height>
    </size>
   </property>
   <property name="font">
    <font>
     <pointsize>20</pointsize>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="titleLabel">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>1200</width>
     <height>100</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>1200</width>
     <height>100</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>1200</width>
     <height>100</height>
    </size>
   </property>
   <property name="font">
    <font>
     <pointsize>30</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>数据库管理</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QTableView" name="tableView">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>160</y>
     <width>761</width>
     <height>561</height>
    </rect>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <property name="autoFillBackground">
    <bool>true</bool>
   </property>
   <property name="styleSheet">
    <string notr="true">QTableView { 
                      		 border: 1px solid black; 
                             padding: 10px; 
                             background: rgba(255, 255, 255, 242); 
                             border-radius: 10px; }</string>
   </property>
  </widget>
  <widget class="QWidget" name="horizontalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>9</x>
     <y>730</y>
     <width>761</width>
     <height>61</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QPushButton" name="prevButton">
      <property name="text">
       <string>&lt;</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="chartshowbtn">
      <property name="text">
       <string>查看图</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="nextButton">
      <property name="text">
       <string>&gt;</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="verticalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>859</x>
     <y>169</y>
     <width>281</width>
     <height>541</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QPushButton" name="addbtn">
      <property name="text">
       <string>增添数据</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="delbtn">
      <property name="text">
       <string>删除数据</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="editbtn">
      <property name="text">
       <string>编辑数据</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QToolButton" name="toolButton">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>20</y>
     <width>28</width>
     <height>26</height>
    </rect>
   </property>
   <property name="text">
    <string>...</string>
   </property>
   <property name="icon">
    <iconset theme="QIcon::ThemeIcon::DocumentProperties"/>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
